### DISCUSSION TITLE
Start review from conversation page

### DISCUSSION THREAD

**Discussion by nazar-pc** [community/community#137579] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

This is the single biggest issue with PRs on GitHub!

There is no way to create or explicitly continue review from "Conversation" tab, similarly it is not possible to create a batch of review suggestions from this tab either.

As the result every single freaking person I deal with for the first time generates an insane number of email notifications as they start replying to my every comment one by one or applying each review suggestion in a separate commit instead of bundling responses as a review so I am not distracted in real time as they review things. This is extremely frustrating!

This is not an issue on GitLab because it allows to create review from equivalent view. This also leads to confusing bugs in UI like Ticket 1713232 that I reported back in 2022.

**Comment by bebuch** [comment]
This is a superset of #36013.

**Comment by bebuch** [comment]
This should be labelled as enhancement.
